angular/src/shared/components/employee/list-employee/list-employee.component.ts

```diff
@@ -300,22 +300,12 @@ export class ListEmployeeComponent extends PagedListingComponentBase<GetEmployee
 
   public onDelete(employee: GetEmployeeDto) {
     const emailConvert = this.extractEmailUsername(employee.email);
```

Comment:
api tùy data mà trả về 1 trong 2 message sau, fe hiển thị message đó ra
 - Deleted employee a.tranvan@ncc.asia successful
 - Deleted both employee and user a.tranvan@ncc.asia

```
this.confirmDelete(`Delete employee <strong>${employee.fullName}</strong> </br> Also delete user <strong>${emailConvert}</strong>` , () => {
      this.employeeService.delete(employee.id).subscribe((result) => {
        abp.notify.success(result.result)
        this.refresh();
      })
```